### PR TITLE
Feature/wr 1075 snowplow module

### PR DESCRIPTION
**Pull request by jchera**
Hi @infojunkie 

I have just made changes to two files:
1. info file - added core and version details to make it work for D9.
2. .module file - Added condition to check the search keys.

### CODE REVIEW

File: src/web/modules/custom/GDX-Analytics-Drupal-Snowplow-main/gdx_analytics_drupal_snowplow.module [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+    default:
+  }
+}
+
+/**
+ * Implements hook_page_attachments().
+ */
+function gdx_analytics_drupal_snowplow_page_attachments(array &$page) {
+  // print 'here'; exit;
```
Comment (infojunkie) [inline src/web/modules/custom/GDX-Analytics-Drupal-Snowplow-main/gdx_analytics_drupal_snowplow.module:32]: please remove debugging code

File: src/web/modules/custom/GDX-Analytics-Drupal-Snowplow-main/gdx_analytics_drupal_snowplow.module [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+  // Get route info and set up link to settings form.
+  $curr_route_name = \Drupal::routeMatch()->getRouteName();
+  $form_route = Url::fromRoute('gdx_analytics_drupal_snowplow.gdx_analytics_drupal_snowplow_settings_form');
+  $link = Link::fromTextAndUrl('Please Configure Your GDX Analytics Drupal Snowplow Module', $form_route);
+  if($curr_route_name == 'search.view_node_search') {
+    $page['#attached']['drupalSettings']['search'] = true;
+  }
+
+// print 'er'; exit;
```
Comment (infojunkie) [inline src/web/modules/custom/GDX-Analytics-Drupal-Snowplow-main/gdx_analytics_drupal_snowplow.module:49]: please remove

File: src/web/modules/custom/GDX-Analytics-Drupal-Snowplow-main/gdx_analytics_drupal_snowplow.module [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+  // Attach the tracking code to front-end pages.
+  if(!$admin_context->isAdminRoute()){
+    if($snowplow_version == 'Standalone Search vE.2.14.0') {
+      if(!empty(Drupal::request()->query->all()['keys'])){
+        $page['#attached']['drupalSettings']['search_terms'] = explode(' ', Drupal::request()->query->all()['keys']);
+      }
+      $page['#attached']['drupalSettings']['script_uri'] = $script_uri;
+      $page['#attached']['library'][] = 'gdx_analytics_drupal_snowplow/gdx_analytics_drupal_snowplow.webtracker_search';
+      // exit;
```
Comment (infojunkie) [inline src/web/modules/custom/GDX-Analytics-Drupal-Snowplow-main/gdx_analytics_drupal_snowplow.module:71]: please remove

File: src/web/modules/custom/GDX-Analytics-Drupal-Snowplow-main/gdx_analytics_drupal_snowplow.module (hunk 1 above)
Comment (jchera) [reply src/web/modules/custom/GDX-Analytics-Drupal-Snowplow-main/gdx_analytics_drupal_snowplow.module]: removed.

File: src/web/modules/custom/GDX-Analytics-Drupal-Snowplow-main/gdx_analytics_drupal_snowplow.module (hunk 2 above)
Comment (jchera) [reply src/web/modules/custom/GDX-Analytics-Drupal-Snowplow-main/gdx_analytics_drupal_snowplow.module]: removed

File: src/web/modules/custom/GDX-Analytics-Drupal-Snowplow-main/gdx_analytics_drupal_snowplow.module (hunk 3 above)
Comment (jchera) [reply src/web/modules/custom/GDX-Analytics-Drupal-Snowplow-main/gdx_analytics_drupal_snowplow.module]: removed

**Review by infojunkie — APPROVED** [review]
Looks good, thanks.

**Pull request merged**